src/components/victory-axis.jsx

```diff
@@ -392,11 +394,11 @@ class VAxis extends React.Component {
             y2={this.tickProperties.y2}
             style={this.style.ticks}/>
           <text x={this.tickProperties.x}
-            y={this.tickProperties.y}
+            y={(this.tickProperties.y - textLengthAndLines[0])}
```

Comment:
get this value from a new function. If you must return several things from a function, use an object with meaningful keys